src/components/Camera/hooks/useDeviceStorageFull.js

```diff
@@ -0,0 +1,36 @@
+import { t } from "i18next";
```

Comment:
When you import `t` like this instead of using the hook, the translations won't change until the app restarts, so if the user changes their language preference, the text in this hook won't change to the new language until restart. Try using the hook instead.